File: src/JavaScript/medium/string/problem2023.js
/**
 * @param {string[]} nums
 * @param {string} target
 * @return {number}
 */
var numOfPairs = function(nums, target) {
    let count = 0
    for(let i = 0; i < nums.length; i++){
        for(let j = i + 1; j < nums.length; j++){
            if(nums[i] + nums[j] == target){
                count++
            }
            if(nums[j] + nums[i] == target){
                count++
            }
        }
    }
    return count
};

let nums = ["777","7","77","77"], target = "7777"
// let nums = ["123","4","12","34"], target = "1234"

numOfPairs(nums, target)

File: src/Python/medium/string/problem2023.py
from typing import List

class Solution:
    def numOfPairs(self, nums: List[str], target: str) -> int:
        count = 0
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                print(nums[i], nums[j])
                if nums[i] + nums[j] == target:
                    count += 1
                if nums[j] + nums[i] == target:
                    count += 1
        return print(count)


nums = ["777","7","77","77"]; target = "7777"

solution = Solution()
solution.numOfPairs(nums, target)